package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1081,13 +1083,51 @@ def get_attr(attrname, default):
         resnames = get_attr('resnames', 'UNK')
         icodes = get_attr('icodes', ' ')
         segids = get_attr('segids', ' ')
+        chainids = get_attr('chainIDs', ' ')
```

Comment:
```suggestion
        chainids = get_attr('chainIDs', '')
```

Your `validate_chainids` function checks for the empty string "", not the space.